modules/acquia_cms_headless/tests/src/Functional/HeadlessHybridModeTest.php

```diff
@@ -68,6 +76,51 @@ public function testAcmsWizardMenu(): void {
       // Ensure that after save button headless mode is enabled.
       $assertSession->pageTextContains('Acquia CMS Pure Headless has been enabled.');
 
+      // Verify administration menus are changes as per headless mode.
+      // Content Menu.
+      $contentMenu = $assertSession->waitForElementVisible('css', '.toolbar-icon-system-admin-content');
+      $contentMenuLi = [
+        'Scheduled Content',
+        'Add content',
+        'Files',
+        'Media',
+        'Scheduled Media',
+      ];
+      $this->childMenuTest($contentMenu, 'Content', $contentMenuLi, '.toolbar-icon-system-admin-content');
```

Comment:
Test here is failing
<img width="913" alt="Screenshot 2023-02-20 at 5 19 31 PM" src="https://user-images.githubusercontent.com/108808209/220117384-ddd29f8c-8451-4deb-8a8a-245391c40631.png">